net/miniupnpd/patches/100-apply-securemode-conf-to-ipv6.patch

```diff
@@ -0,0 +1,22 @@
+From 45131e0c7c084409aa52d1b7f7408d9d5fecb72e Mon Sep 17 00:00:00 2001
+From: Self-Hosting-Group
+ <155233284+Self-Hosting-Group@users.noreply.github.com>
+Date: Mon, 1 Jan 2024 00:00:00 +0000
+Subject: [PATCH] miniupnpd: Apply `secure_mode` config to IPv6 as well
+
+With IPv6 and UPnP IGD it could not be disabled before
```

Comment:
I can not see that this patch was submitted to the upstream.

Reply:
Why I am already getting third notification that you respond to my comment, then you delete it, and then you write something else and that goes one more round?

I dont see why we should have this patch as upstream rejected it:
https://github.com/miniupnp/miniupnp/pull/678

Thus no, this patch is not getting in.

Reply:
Your pull request is already huge, please do it how it should be and then it is possible to review it. Otherwise, it will take a lot of time to go through all of it.

Reply:
@BKPepe this PR looks more or less compliant now. The commit messages give reason to the changes.